roles/base/tasks/redhat_tasks.yml

```diff
@@ -22,3 +23,17 @@
 
 - name: install and start ntp
   service: name=ntpd state=started enabled=yes
+
+- name: enable NFS server
```

Comment:
@erikh I know this adds more work for you but we should add these tasks in a `nfs` role and add them as dependency for volplugin role (just like we have done for `ceph-install` [role here](https://github.com/contiv/ansible/blob/master/roles/contiv_storage/meta/main.yml#L2)). That way we can keep `base` role lean and keep volplugin's dependency close to it.